spec/migrations/20191002103406_remove_quadicon_settings_spec.rb

```diff
@@ -0,0 +1,17 @@
+require_migration
+
+describe RemoveQuadiconSettings do
+  migration_context :up do
+    let(:user_stub) { migration_stub(:User) }
+
+    it "removes the quadicon from user settings" do
+      u = user_stub.create!(:settings => { :quadicons => { :foo => :bar } })
```

Comment:
You might want a second key in here just to verify that the only thing removed was `:quadicons` and nothing else was touched

Reply:
and that it doesn't change users w/o quadicon in their keys